### DISCUSSION TITLE
REM interface blank and no control after updating to v7.5

### DISCUSSION THREAD

**Discussion by Exit2Studios** [tagyoureit/nodejs-poolController#516] [App and O/S Config Questions] [answered]
My REM web interface is completely blank and I have no control over anything since upgrading REM.

<img width="1428" alt="image" src="https://user-images.githubusercontent.com/16908875/177421211-6b2f69d7-e088-4d09-9bf0-6d72f7152ac4.png">

**Comment by tagyoureit** [comment]
a) Try a hard refresh.  Ctrl-F5 or Shift-F5 or Ctrl-Reload or Ctrl+Shift+R depending on your browser.
and 
b) if that doesn't work, look in the Javascript developer console and there will likely be an error there.  Past it (and a screenshot) here.

**Comment by Exit2Studios** [comment]
I cleared my browsing history and now it won't come up at all lol.

**Comment by Exit2Studios** [comment]
This seems strange however:
<img width="565" alt="image" src="https://user-images.githubusercontent.com/16908875/177426375-31f1c0c8-33a3-4312-9a01-8bc7bab35363.png">

**Reply by tagyoureit** [reply]
`pm2 logs REM` and see what error is happening.

**Comment by Exit2Studios** [comment]
<img width="568" alt="image" src="https://user-images.githubusercontent.com/16908875/177426462-c7e7ec8c-7e1d-4eed-bf43-9ab9ab11afa7.png">

**Comment by Exit2Studios** [comment]
<img width="776" alt="image" src="https://user-images.githubusercontent.com/16908875/177568945-d5c7d3b3-b18a-46c1-ab7a-f86603164cff.png">

**Comment by Exit2Studios** [comment, marked as answer]
I solved it using the following:

`  rm -rf node_modules/robotjs

  npm install`

**Reply by tagyoureit** [reply]
It looks like you upgraded from Node v12 to Node v14.  Anytime you upgrade node versions you should blow away the entire node_modules directory and re-install.  Glad you figured it out!
